extensions/amp-story/1.0/amp-story-draggable-drawer.js

```diff
@@ -556,6 +620,15 @@ export class DraggableDrawer extends AMP.BaseElement {
         this.mutateElement(() => resetStyles(this.element, ['transition']));
       }
 
+      if (
+        isPageAttachmentUiV2ExperimentOn(this.win) &&
+        this.element.tagName === 'AMP-STORY-PAGE-ATTACHMENT'
+      ) {
+        this.element
```

Comment:
We have a `closest()` method on AMP that is more widely used than the `element.closest` and you can pass in the check function (in this case would be `tagName === "..."`), the limit (which could be the amp-story) and `this.element`.